### DISCUSSION TITLE
Unable to read HiveServer2 configs from ZooKeeper

### DISCUSSION THREAD

**Discussion by merrily01** [apache/kyuubi#4436] [Q&A] [answered]
1. Using the zookeeper mode, the server configuration is as follows: 
```
kyuubi.ha.enabled                                        true
kyuubi.ha.zookeeper.quorum                      node1:2181,node2:2181,node3:2181
kyuubi.ha.zookeeper.client.port                 2181
kyuubi.ha.zookeeper.namespace               /zookeeper/kyuubi
```
2. Kyuubi server is started and successfully registered in zk : 

![image](https://user-images.githubusercontent.com/15688792/222351654-3fdf1cb1-f87c-4584-bd96-ece079528553.png)

4. The connection can be successful connected through `beeline kyuubi server ip: port`, but an error is reported through ```./beeline -u "jdbc:hive2://node1:2181,node2:2181,node3:2181/default;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=/zookeeper/kyuubi"  -n test```,  as follows:
```
Error: org.apache.kyuubi.jdbc.hive.ZooKeeperHiveClientException: Unable to read HiveServer2 configs from ZooKeeper (state=,code=0)
```

**There are few logs and the prompts are vague. What may be the problem?**

**Comment by pan3793** [comment, marked as answer]
Try change "zooKeeperNamespace=/zookeeper/kyuubi" to "zooKeeperNamespace=zookeeper/kyuubi", remove the leading slash

**Reply by pan3793** [reply]
And please provide the whole stacktrace, a one-line error message is not much help to finding the root cause sometimes.

**Reply by merrily01** [reply]
Thanks for the response, it is effective. @pan3793 Thanks♪(･ω･)ﾉ

**Reply by pan3793** [reply]
I think we can tolerate such a case, would you like to send a PR to fix it?

We can strip the leading and tailing slash of `ZooKeeperHiveClientHelper#getZooKeeperNamespace`'s return value.

**Reply by merrily01** [reply]
Yeah, of course.
I will work on this later. Thx~
